requirements/requirements-cirq.txt

```diff
@@ -0,0 +1,3 @@
+# This file is redundant, since the internal representation of choice is Cirq.
```

Comment:
I would rephrase just to make this a little clearer. Something like, "...since Cirq is the core internal representation for quantum circuits in Mitiq"

Reply:
Yes, better